src/main/java/com/google/sps/servlets/GetElectionServlet.java

```diff
@@ -0,0 +1,169 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.cloud.secretmanager.v1.AccessSecretVersionResponse;
+import com.google.cloud.secretmanager.v1.SecretManagerServiceClient;
+import com.google.cloud.secretmanager.v1.SecretVersionName;
+import com.google.gson.Gson;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+@WebServlet("/getElection")
+
+/**
+ * This servlet is used to update the gVote database with information on the ongoing elections that
+ * an eligible voter can participate in on a given day.
+ */
+public class GetElectionServlet extends HttpServlet {
+
+  private static final String baseURL = "https://www.googleapis.com/civicinfo/v2/elections?key=";
+  private static final Logger logger = Logger.getLogger(ElectionServlet.class.getName());
+
+  // This method is used to access the api key stored in gcloud secret manager.
+  public String accessSecretVersion(String projectId, String secretId, String versionId)
+      throws IOException {
+    try (SecretManagerServiceClient client = SecretManagerServiceClient.create()) {
+      SecretVersionName secretVersionName = SecretVersionName.of(projectId, secretId, versionId);
+
+      // Access the secret version.
```

Comment:
Remove this comment. This is an example of a "bad" comment which describes the "what I'm doing" not the "why I'm doing it". I can just read the line below and figure out from "client.accessSecretVersion()" what the comment is saying. 

You could replace this comment explaining why you don't want to reveal the API key and instead want to store it in secret manager (e.g. because ppl scrape github for API keys, etc. etc.)